mesonbuild/backend/ninjabackend.py

```diff
@@ -29,6 +29,7 @@
 
 from . import backends
 from .. import modules
+from ..modules import gnome
```

Comment:
```
modules: move gnome targets into gnome module

They're not used outside of the gnome module anyway, and they create
some annoying potentials for dependency loops
```

... except when they are, of course. :P I just rediscovered this commit while reducing imports everywhere. Not only does this drag a 2k line file into being unconditionally imported at startup, it also indirectly drags in PkgConfigDependency which I've been in the process of ripping out.

Looking over this again, it's not obvious to me what the dependency loop here is?